awscli/customizations/eks/get_token.py

```diff
@@ -61,11 +62,18 @@ def _run_main(self, parsed_args, parsed_globals):
             parsed_args.role_arn
         )
 
+        # By default STS signs the url for 15 minutes so we are creating a
+        # rfc3339 timestamp with expiration in 14 minutes as part of the token, which
+        # is used by some clients (client-go) who will refresh the token after 14 mins
+        token_expiration = datetime.utcnow() + timedelta(minutes=14)
```

Comment:
Can you make the 14 a constant?

Also https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/160/files has a lot of things going on. Can you make sure we are doing something similar? So that we dont risk introducing more unexpected things

Reply:
I will update it to a constant.

Yeah we seem to have covered whatever aws-iam-authenticator is doing on the client side.